<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgSetup</class>
 <widget class="QDialog" name="DlgSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CocosPlaySDK</string>
  </property>
  <widget class="QPushButton" name="btnSetup">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>320</y>
     <width>138</width>
     <height>53</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border:none;background-image: url(:/btnInstallNormal);color:rgb(255,255,255);}
QPushButton:hover{background-image: url(:/btnInstallHover);}
QPushButton:pressed{background-image: url(:/btnInstallPress);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="CScale9Image" name="imgBg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>698</width>
     <height>604</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="imgBackground">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>680</width>
     <height>586</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="data.qrc">:/background</pixmap>
   </property>
  </widget>
  <widget class="QFrame" name="hFrameDir">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>510</y>
     <width>698</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="txtDir">
      <property name="styleSheet">
       <string notr="true">color:rgb(71,71,71);</string>
      </property>
      <property name="text">
       <string>安装位置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditFile">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>24</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{border:1px solid rgb(176,181,187);background-color:rgb(255,255,255);}
QLineEdit:hover{border:1px solid rgb(131,245,253);}</string>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnbrowse">
      <property name="minimumSize">
       <size>
        <width>69</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>69</width>
        <height>28</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{border:none;background-image: url(:/btnWhiteMiddleNormal);}
QPushButton:hover{background-image: url(:/btnWhiteMiddleHover);}
QPushButton:press{background-image: url(:/btnWhiteMiddlePress);}</string>
      </property>
      <property name="text">
       <string>浏览</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnMinimize">
   <property name="geometry">
    <rect>
     <x>623</x>
     <y>10</y>
     <width>27</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>27</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>27</width>
     <height>23</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#btnMinimize{border:none;background-image: url(:/minimize);}
#btnMinimize:hover{background-image: url(:/minimize_hover);}
#btnMinimize:pressed{background-image: url(:/minimize_press);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>27</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>27</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>27</width>
     <height>23</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#btnClose{border:none;background-image: url(:/clouse);}
#btnClose:hover{background-image: url(:/clouse_hover);}
#btnClose:pressed{background-image: url(:/clouse_press);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>190</y>
     <width>520</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="data.qrc">:/logo</pixmap>
   </property>
  </widget>
  <widget class="QFrame" name="hFrameProgress">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>340</y>
     <width>698</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="gifInstalling">
      <property name="minimumSize">
       <size>
        <width>92</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnFinish">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>340</y>
     <width>138</width>
     <height>53</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border:none;background-image: url(:/btnStartNormal);}
QPushButton:hover{background-image: url(:/btnStartHover);}
QPushButton:pressed{background-image: url(:/btnStartPress);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="hFrameShowFinish">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>698</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="txtInstallFinish">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="data.qrc">:/imgInstallFinish</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="imgInstallTxt">
   <property name="geometry">
    <rect>
     <x>123</x>
     <y>470</y>
     <width>186</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="data.qrc">:/imgInstallTxt</pixmap>
   </property>
  </widget>
  <zorder>imgBg</zorder>
  <zorder>imgBackground</zorder>
  <zorder>hFrameDir</zorder>
  <zorder>btnMinimize</zorder>
  <zorder>btnClose</zorder>
  <zorder>label</zorder>
  <zorder>hFrameProgress</zorder>
  <zorder>hFrameShowFinish</zorder>
  <zorder>imgInstallTxt</zorder>
  <zorder>btnSetup</zorder>
  <zorder>btnFinish</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CScale9Image</class>
   <extends>QLabel</extends>
   <header>cscale9image.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="data.qrc"/>
 </resources>
 <connections/>
</ui>
